Update of a few configuration examples and related documentation in the readme file

venturia opened the pull request.

cmsbuild commented:
A new Pull Request was created by @venturia  for CMSSW_7_5_X.

Update of a few configuration examples and related documentation in the readme file

It involves the following packages:

CalibTracker/SiStripQuality


@cmsbuild, @mmusich, @diguida, @cerminar, @nclopezo can you please review it and eventually sign? Thanks.
@jlagram, @gbenelli this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@nclopezo you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

diguida commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1 
Tested at: 6a63df03a19fd6dec9578e523135c9d17ccb0b2f
 When I ran the RelVals I found an error in the following worklfows: 
 25.0 step4
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT/step4_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT.log

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8569/3037/summary.html

cmsbuild committed:
Merge pull request #8655 from emanueledimarco/egamma_full5x5mva_740pre9

Use of full5x5 cluster shapes to compute the MVA, plus other flexibility

cmsbuild committed:
Merge pull request #8654 from lveldere/fastsim-fix-recoTrack-truthmatching-putracks-75X

Fastsim 75X: fix truth matching for recoTracks from PU events

cmsbuild committed:
Merge pull request #8493 from gzevi/pat_photon_isolation_conflict_with_reco4

prevent PAT photon iso variables from overwriting Reco ones

Commit:
Merged nSigmaZSelection from repository lveldere

Commit:
move changes to HLT menu to FastSim area (temporary measure)

Commit:
remove obsolete test files

BetterWang committed:
[-] remove unused comments

Commit:
fix several errors introduced during merging and rebasing

mileva committed:
Updated to GEM didi model from SLHC23

mileva committed:
Updated to GEM didi model from SLHC23, commented lines removed

Commit:
removed dead cfg file, import FastSim version of MeasurementTrackerEvent cfg in Reconstruction_BefMix

Commit:
fix merge error in HLT cfg

emanueledimarco committed:
disable the Leading Edge recovery in both multifit and global uncalibrated rechit producers

emanueledimarco committed:
make the test cfg work in late 75X

Commit:
CastorDigiProducer : make input tag for SimHits configurable

Commit:
Merged refs/pull/8695/head from repository cms-sw

Commit:
fixed mistake

Commit:
manually moving files for GeneratorInterface/EvtGenInterface/ from inugent:BSTomumu_Patch3 to new branch

wddgit committed:
Consumes migration for BCToEFilter

Add consumes call in BCToEFilter. Also change
getByLabel to getByToken, fix a minor memory
leak, and clean up the header includes a bit.

cmsbuild committed:
Merge pull request #8695 from lveldere/castordigiproducer-simhittag

CastorDigiProducer : make input tag for SimHits configurable

capalmer85 committed:
pcc-ntupler-74X-v1

cmsbuild committed:
Merge pull request #8694 from mileva/from-CMSSW_7_5_X_2015-04-09-2300_BackToGemDigiModel

From cmssw 7 5 x 2015 04 09 2300 back to gem digi model

cmsbuild committed:
Merge pull request #8673 from wmtford/MCsplitStrips

Strip cluster splitter (MC truth based)

cmsbuild committed:
Merge pull request #8699 from inugent/BSTomumu_Patch4

manually moving files for GeneratorInterface/EvtGenInterface/ from inuge...

cmsbuild committed:
Merge pull request #8665 from BetterWang/ZDC_onlineDQM_75X

ZDC DQM update for 75X

pietverwilligen committed:
Merged refs/pull/8524/head from repository cms-sw

Commit:
add an 'Empty SimHitCollection' producer, write a customisation function that adds empty SimHit collections for detector parts that were not simulated

Commit:
FastSim define customisation function to disable material interactions in tracker

Commit:
change a confusing function name: fakeSimHits_for_geometry_HCALECAL -> fakeSimHits_for_geometry_ECALHCAL

cmsbuild committed:
Merge pull request #8649 from bsunanda/Run2-alca7

bsunanda:Run2-alca7 Correct the destructors and a few constructors and a few python scripts ...

Commit:
Merged nSigmaZSelection from repository lveldere

emanueledimarco committed:
saturation: use the extrapolation from the 4th sample only if thesaturation happens on the 5th. Use the 5th sample measurement otherwise, to give an estimate of the in-time energy

Commit:
convert fastsim addon tests to new cfg structure

Commit:
let cmsDriver.py spit a useful message when no input file is given for a combination of steps that requires one

Commit:
susy wfs SMS-T1tttt_mGl-1500_mLSP-100_13 geared towards fastsim

pietverwilligen committed:
commented in LogDebug and removed dynamic casts

pietverwilligen committed:
removed unnecessary comments

pietverwilligen committed:
created to run CSC+DT+RPC only digitization

pietverwilligen committed:
test RPC Digitization in different geometries

Commit:
dont do gen-sim recycle guesses

pietverwilligen committed:
RPC upgrade :: ready to go up to 192 strips / roll

pietverwilligen committed:
GEM upgrade :: run all muon detectors local reco

pietverwilligen committed:
Muon upgrade :: fix condition problems

pietverwilligen committed:
GEM upgrade :: run all muon detectors digi step

pietverwilligen committed:
RPC Digitizer :: give RPCGeometry to SimSetup

pietverwilligen committed:
RPC Digitizer :: ready to go up to 192 strips / roll (removed hardcoded 96 strips / roll in the reading of the conditions

pietverwilligen committed:
RPC Digitizer :: test all muon detectors digi step for different geometries

pietverwilligen committed:
RPC Digitizer :: customize to run digitization only for muon detectors if required (so it is more easy to debug RPC code when also other detectors are changing their code

pietverwilligen committed:
RPC Digitizer :: test all muon detectors digi step for different geometries

pietverwilligen committed:
GEM upgrade :: run all muon detectors digi step

pietverwilligen committed:
GEM upgrade :: run all muon detectors local reco

pietverwilligen committed:
Muon upgrade :: fix condition problems

pietverwilligen committed:
Merge branch 'RPCDigitizerForUpgrade75XPR' of https://github.com/pietverwilligen/cmssw into RPCDigitizerForUpgrade75XPR

Conflicts:
	RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py
	SimMuon/GEMDigitizer/test/runGEMDigiProducer_cfg.py
	SimMuon/RPCDigitizer/test/runRPCDigiProducer_cfg.py

pietverwilligen committed:
removed the for the missing upgrade GT payloads to separate file

emanueledimarco committed:
consider that the histeresis of the ADC only lasts for max 5 samples

Commit:
config files to run MultiTrackValidator on FastSim samples

Commit:
Merged fastsim-MTV from repository lveldere

Commit:
undo non-essential changes to simplify pr review procedure

Commit:
fix output path of FastSimulation/Validation/test/MultiTrackValidator_cff.py

wmtan committed:
Macros need to compile in ROOT6

wmtan committed:
Macros need to compile in ROOT6

wmtan committed:
Macros need to compile in ROOT6

wmtan committed:
Macros need to compile in ROOT6

wmtan committed:
Macros need to compile in ROOT6

wmtan committed:
Macros need to compile in ROOT6

pastika committed:
Replaced old geometry files with correct geometry file

wmtan committed:
Macros need to compile in ROOT6

cmsbuild committed:
Merge pull request #8704 from shchauha/nSigmaZSelection_final

Streamlining FastSim tracking (cuts & quality flags) along reco tracking, adapt to reco tracking naming scheme of collections

cmsbuild committed:
Merge pull request #8619 from lihux25/fix_hcal_dump_conditions_topo_issue

Fix hcal dump conditions topo issue

cmsbuild committed:
Merge pull request #8472 from apfeiffer1/listGTsForTag

List GTs for a given tag in conddb

cmsbuild committed:
Merge pull request #8447 from igv4321/negative-energy-filter-fix

Database object for the HCAL negative energy filter

davidlange6 committed:
Merge pull request #8657 from pastika/7_5_X_HCALRelval_Update_1127_3_4_2015

Updates to HCAL relval scripts and standalone configs

davidlange6 committed:
Merge pull request #8701 from wddgit/consumes102

Consumes migration for BCToEFilter

cmsbuild committed:
Merge pull request #8708 from franzoni/addSusyCMSSW_7_5_0_pre2_FETCHED

new susy GEN configuration

cmsbuild committed:
Merge pull request #8601 from amkalsi/DQM-Files-Code-forVBF-SUSY-MuHLTPath

Dqm files code for vbf susy mu hlt path

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Commit:
Merged refs/pull/8703/head from repository cms-sw

Commit:
moved content of FastSimulation/EmptySimHits to FastSimulation/Validation

davidlange6 committed:
forward port from 73x

davidlange6 committed:
Merge pull request #8718 from davidlange6/hcalnzRun2

add run2 hcalzns config

pietverwilligen committed:
Merge pull request #1 from cms-sw/CMSSW_7_5_X

PRToFixMess_RPCDigitizerForUpgrade75XPR

Commit:
finialised move of EmptyHits to FastSimulation/Validation

quertenmont committed:
fix calibration for 75X

geoff-smith committed:
new 75X branch for hlt integrations

ianna committed:
Use HcalTopology mode as defined in SLHC releases.

wddgit committed:
Consumes migration for AlignmentTrackSelectorModule

Add consumes calls for module AlignmentTrackSelectorModule.
Also convert getByLabel calls to getByToken. These calls
were a couple levels deep in 3 helper classes.

wddgit committed:
Consumes migration for CosmicGenFilterHelix

Add consumes function calls for the module
CosmicGenFilterHelix. Convert getByLabel call
to getByToken.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Commit:
changing the HBHE filter used by offline DQM

cmsbuild committed:
Merge pull request #8715 from wmtan/macrosReco

Macros need to compile in ROOT6 (Reco)

cmsbuild committed:
Merge pull request #8710 from wmtan/macrosALCA

Macros need to compile in ROOT6 (ALCA)

cmsbuild committed:
Merge pull request #8709 from wmtan/macrosSim

ROOT Macros need to compile in ROOT6 (Sim)

Commit:
Merged refs/pull/8748/head from repository cms-sw

pietverwilligen committed:
removed double setup of the RPCGeometry

pietverwilligen committed:
Merge pull request #2 from cms-sw/CMSSW_7_5_X

PRToFixMess_RPCDigitizerForUpgrade75XPR_v2

pietverwilligen committed:
Merge branch 'RPCDigitizerForUpgrade75XPR' of https://github.com/pietverwilligen/cmssw into RPCDigitizerForUpgrade75XPR

geoff-smith committed:
cmshlt-310 paths add to GRun, 50nsGRun

cmsbuild committed:
Merge pull request #8721 from arizzi/fixPVassoAbsBug-from-CMSSW_7_4_0

Fix bug in PVAssignment: missing abs for dxy

cmsbuild committed:
Merge pull request #8705 from lveldere/fastsim-addontests-new-cfg

convert fastsim addon tests to new cfg structure

cmsbuild committed:
Merge pull request #8688 from elaird/CMSSW_7_5_X

add feds

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #8727 from ianna/hcal-topology-parameter-as-string

Use Hcal Topology Mode as Defined in SLHC Releases

Commit:
add a equivalent of Validation/RecoTracker/test/MultiTrackValidator_cfg.py to FastSim/Validation/test

emanueledimarco committed:
check if the wrong gain transition is kept for 5 consecutive samples or not

emanueledimarco committed:
use an existing file for test

pietverwilligen committed:
added LogDebug

pietverwilligen committed:
added LogDebug

pietverwilligen committed:
added LogDebug

pietverwilligen committed:
added LogDebug

pietverwilligen committed:
fixed bug: noise of strip 1 was loaded twice in noisevector

pietverwilligen committed:
implementation of LogDebug and added tests on Run 1 MC

Commit:
add scripts to validation tracks, seeds and tracking mc truth for fastsim

Commit:
Merged refs/pull/8748/head from repository cms-sw

Commit:
FastSimulation/Validation/test/: removed obsolete file MultiTrackValidator_cff.py, adapted input file names for TrackRecoAndTrackValidation_cfg.py and MultiTrackValidator_cfg.py

pietverwilligen committed:
added LogDebug to check for first Random Number in RPC Digitizer

pietverwilligen committed:
make sure to keep RPC digi collection

Martin-Grunewald committed:
Update for new FastSim treatment

Martin-Grunewald committed:
Fixup of tests

Commit:
Changed default HBHE noise filter flag for Run 2 data processing

cmsbuild committed:
Merge pull request #8759 from emanueledimarco/ecalunpacker_fixgainswitch_750pre2

ECAL unpacker: consider the ADC hysteresis in the gain switches

cmsbuild committed:
Merge pull request #8754 from rappoccio/rappoccio_FixSubjetIndex74x

Fixing reversed distance in accessing subjet collections

cmsbuild committed:
Merge pull request #8714 from wmtan/macrosAnal

Macros need to compile in ROOT6 (Analysis)

cmsbuild committed:
Merge pull request #8713 from wmtan/macrosCommonTools

Macros need to compile in ROOT6 (CommonTools)

cmsbuild committed:
Merge pull request #8690 from deguio/fixDQMtests

fix broken DQM tests

davidlange6 committed:
Merge pull request #8678 from bmarzocc/7_5_X_alcarecostream_nophisym

ALCARECOSTREAM_nophisym

Martin-Grunewald committed:
More FastSim treats

Dr15Jones committed:
Made PileupJetIdProducer into a stream module

Profiling showed that this module was leading to inefficiency when
running DQM in threaded mode.

Commit:
rename TrackRecoAndTrackValidation_cfg.py, add a FullSim equivalent of TrackRecoAndValidation_cfg.py

apfeiffer1 committed:
conddb: show size of payloads in search (still to be verified); switch to no-color mode if stdout is pipe

pietverwilligen committed:
more information on how to use the LogDebug

pietverwilligen committed:
Merge pull request #3 from cms-sw/CMSSW_7_5_X

PRToFixMess_RPCDigitizerForUpgrade75XPR_v3

Commit:
more fastsim hlt changes

Martin-Grunewald committed:
More FastSim

Commit:
restored a line in confdb.py to redefine GMTReadoutCollection for FastSim

Martin-Grunewald committed:
More FastSim fixes

emanueledimarco committed:
remove the extra S5/S4 ratio for the cases where the saturation is in sample 5 and just put the saturated value

Dr15Jones committed:
No longer need AutoLibraryLoader

With ROOT 6 we no longer use the AutoLibraryLoader to load dictionaries. We now allow ROOT to do the loading itself.

Martin-Grunewald committed:
Further fixes in FastSim

apfeiffer1 committed:
fix help for new nocolors option

cmsbuild committed:
Merge pull request #8775 from silviodonato/HLT2PhotonMET

Adding of HLT2PhotonMET

cmsbuild committed:
Merge pull request #8766 from Dr15Jones/threadSafePileupJetIdProducer

Made PileupJetIdProducer into a stream module

cmsbuild committed:
Merge pull request #8748 from lveldere/fastsim-MTV

configs to run MultiTrackValidator standalone for FastSim

cmsbuild committed:
Merge pull request #8778 from mmarionncern/PFCalib-74X

 Update PF hadron calibration code (same as #8773 from 74X)

cmsbuild committed:
Merge pull request #8758 from deguio/fixTrackingT0

fix typo in the definition of the T0 sequences for tracking

cmsbuild committed:
Merge pull request #8717 from emanueledimarco/disable_LErecovery_750p2

Disable Not Working leading edge recovery for saturated ECAL RecHits

cmsbuild committed:
Merge pull request #8712 from wmtan/macrosVis

Macros need to compile in ROOT6 (Visualization)

Commit:
fix segfault: new edalias caused same branch to be stored twice: combination of related error message and unresolved bug in HFShowerLibrary destructor caused segfault

Martin-Grunewald committed:
Adding fix from Lukas on event content leading to a crash

Martin-Grunewald committed:
More FastSim fixes

Martin-Grunewald committed:
More FastSim fixes

cmsbuild committed:
Merge pull request #8782 from Dr15Jones/updateClassVersionCheck

No longer need AutoLibraryLoader for edmCheckClassVersion

cmsbuild committed:
Merge pull request #8769 from apfeiffer1/conddb-nocol

conddb improvements

cmsbuild committed:
Merge pull request #8691 from rovere/RunIRecoTuning

Run 1 reco tuning

Commit:
Merged refs/pull/8753/head from repository cms-sw

Commit:
added line to confdb.py to replace InputTag(hltIter4HighPtMerged) with InputTag(generalTracks)

ianna committed:
Axes in DDI namespace.

Martin-Grunewald committed:
Merged hlt-fastsim-collection-edits from repository lveldere

Commit:
add tau-sequence to fastsim DQM plot harvesting

Commit:
remove white-space

Martin-Grunewald committed:
Swap fix to be in correct file

Dr15Jones committed:
Fix compilation error in EventSetup::tryToGet

A typo in EventSetup::tryToGet kept the templated function from compiling. Added a unit test to make sure the problem does not reoccur.

wddgit committed:
Fix errors reported by valgrind

Fix errors reported by valgrind related
to uninitialized variables. Really the
same error repeated in 3 places.

capalmer85 committed:
technical updates:  getByToken, edm::one::EDAnalyzer<> (multi-thread aware)

cmsbuild committed:
Merge pull request #8794 from ianna/dd-axes-in-ddi-namespace

Axes in DDI Namespace

cmsbuild committed:
Merge pull request #8668 from wddgit/consumes101

Consumes migration

pietverwilligen committed:
commented line in LogDebug where RandFlat was fired

pietverwilligen committed:
commented line in LogDebug where RandFlat was fired

pietverwilligen committed:
Merge pull request #4 from cms-sw/CMSSW_7_5_X

PRToFixMess_RPCDigitizerForUpgrade75XPR_v4

pietverwilligen committed:
Merge branch 'RPCDigitizerForUpgrade75XPR' of https://github.com/pietverwilligen/cmssw into RPCDigitizerForUpgrade75XPR

davidlange6 committed:
Merge pull request #8802 from Dr15Jones/fixEventSetupTryToGet

Fix compilation error in EventSetup::tryToGet

davidlange6 committed:
Merge pull request #8751 from slava77/CMSSW_7_4_0/csc-me11A-emulator

bug fixes in CSC (ME11) trigger primitives for re-emulation; fix ME11 selective readout (fw port of 73X #8664 )

quertenmont committed:
fixes from Marco M.

cmsbuild committed:
Merge pull request #8685 from igv4321/hcal-noise-filters-update

Hcal noise filters update

makortel committed:
Remove old-style track associator ESProducer configuration files

cmsbuild committed:
Merge pull request #8703 from lveldere/EmptySimHits

Enable standard digitisation and reconstruction on simulation HCALECAL geometry

ianna committed:
Remove depricated files.

ianna committed:
Add warning about using Run 1 transient geometry.

capalmer85 committed:
add lumi section awareness

vkhristenko committed:
HCAL DQM uTCA-related changes

capalmer85 committed:
add override

wmtan committed:
Avoid segfault when parentage ID is out of bounds

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

wmtan committed:
Use correct type for consumes declaration

wmtford committed:
Clear maps each new event

davidlange6 committed:
Merge pull request #8753 from cms-tsg-storm/hltIntegration75X

continuing HLT integrations - 75X

cmsbuild committed:
Merge pull request #8804 from Sam-Harper/JetHLTMatching74X

allowing E/g HLT to be seeded by L1 jets

vkhristenko committed:
Adding HO unmaskinng

vkhristenko committed:
Addin HO unmasking

Commit:
clearn way to remove iterative tracking modules from reconstruction sequence

Commit:
revert a few changes

Commit:
removing dead files from FastSimulation/Configuration

Commit:
more cleanups of Reconstruction_AftMix_cff.py

Commit:
remove obsolete dependence of FamosSimHits_cff on CommonInputs_cff

davidlange6 committed:
Merge pull request #8810 from ianna/geom-config-cleanup

Geometry Configuration Cleanup

Dr15Jones committed:
Better error message for bad SequencePlaceholder

If a SequencePlaceholder is referring to an object of a type which is
not allowed on a Sequence (e.g. EDAlias) we now issue a useful exception
message.

Dr15Jones committed:
Process.prune can ignore unresolved SequencePlaceholders

In order to allow prune to be called on a partially completed Process, there is now an optional argument for prune which will allow unresolved SequencePlaceholders from causing an exception.

ianna committed:
Use strongly-typed enum.

Commit:
remove dead package FastSimulation/CaloRecHitsProducer

Commit:
remove dead package  FastSimulation/L1CaloTriggerProducer/

cmsbuild committed:
Merge pull request #8827 from Dr15Jones/betterErrorMessageForBadSequencePlaceholder

Better error message for bad SequencePlaceholder

davidlange6 committed:
Merge pull request #8731 from wddgit/consumesSelector

Consumes migration for AlignmentTrackSelectorModule

cmsbuild committed:
Merge pull request #8815 from wmtan/AvoidSegfault

Avoid segfault in edmProvDump -d when parentage ID is out of bounds

Commit:
Switching 50 ns Run 2 configuration of HCAL noise filters to Run 1 mode

wmtford committed:
Clear maps in the deprecated makeMaps also

Dr15Jones committed:
Force use of OscarMTProducer when running multi-threaded

Until we switch to using OscarMTProducer for all simulations we need to have the IBs which run multi-threaded use OscarMTProducer. The multi-threaded IBs all call dropNonMTSafe to turn on threading so this hack will do the trick for now.

Commit:
fix issues reported by static analyzer

cmsbuild committed:
Merge pull request #8836 from Dr15Jones/forceUseOfOscarMTProducerWhenMultithreading

Force use of OscarMTProducer when running multi-threaded

davidlange6 committed:
Merge pull request #8734 from wddgit/consumesCosmicGenFilterHelix

Consumes migration for CosmicGenFilterHelix

cmsbuild committed:
Merge pull request #8816 from wmtan/UseCorrectTypeForConsumes

Use correct type for consumes declaration

cmsbuild committed:
Merge pull request #8809 from makortel/trackAssociationCleanup2

Remove old-style track associator ESProducer configuration files

davidlange6 committed:
Merge pull request #8832 from deguio/noRezise

avoid having axis resized for LogMessageMonitor

davidlange6 committed:
Merge pull request #8716 from wmtan/macrosDQM

Macros need to compile in ROOT6 (DQM)

davidlange6 committed:
Merge pull request #8819 from wmtford/assocClearMaps

TrackerHitAssociator clear maps each new event

Commit:
The script dqmdata_cleaner.py is updated and pushed to dqmgui repo where it belongs.

ianna committed:
Use correct geometry.

cmsbuild committed:
Merge pull request #8524 from pietverwilligen/RPCDigitizerForUpgrade75XPR

Rpc digitizer for upgrade75 xpr

Martin-Grunewald committed:
Forward Port Part I

davidlange6 committed:
Merge pull request #8829 from ianna/dd-strongly-typed-enum

Clean up DD Axes

davidlange6 committed:
Merge pull request #8849 from ianna/use-correct-geometry-in-reco-met

Use Correct Geometry in RecoMET Unit Test

cmsbuild committed:
Merge pull request #8830 from lveldere/fastsim-remove-dead-packages

Fastsim remove dead packages

cmsbuild committed:
Merge pull request #8799 from lveldere/fastsim-tauharvesting

Fastsim: 75X:  use same tau DQM plot harvester as FullSim

davidlange6 committed:
Merge pull request #8828 from Dr15Jones/allowProcessPruneToIgnoreUnresolvedSequencePlaceholders

Process.prune can ignore unresolved SequencePlaceholders

davidlange6 committed:
Merge pull request #8846 from batinkov/CMSSW_7_5_X

The script dqmdata_cleaner.py is deleted.

Martin-Grunewald committed:
Forward port Part II

vkhristenko committed:
Making HF seen as present with VME FEDs

davidlange6 committed:
Merge pull request #8851 from Martin-Grunewald/ForwardPort

HLT fixes and cfg rejig (forward port of #8781)

capalmer85 committed:
add consumes for pileup info

davidlange6 committed:
Merge pull request #8679 from VinInn/SelTracks

upgrade RecoTrackSelector to allow BTV-like cuts

cmsbuild committed:
Merge pull request #8803 from wddgit/from-CMSSW_7_5_X_2015-04-20-1100

Fix errors reported by valgrind

cmsbuild committed:
Merge pull request #8726 from quertenmont/FixGainCalibration

fix gain calibration files for 75X

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Commit:
fixed Reconstruction_AftMix_cff

davidlange6 committed:
Merge pull request #8702 from capalmer85/pcc-ntupler-74X-v1

pcc-ntupler-74X-v1

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

davidlange6 committed:
Merge pull request #8842 from arizzi/fixPVforPFEle-from-CMSSW_7_4_0_pre9

MINIAOD BUGFIX: pvREF not set for some electrons

alja committed:
Merged fwdict6-74 from repository alja

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #8854 from lveldere/fastsim-reco-cleanup

Fastsim reco cleanup

cmsbuild committed:
Merge pull request #8833 from igv4321/hcal-noise-filters-config

Switching 50 ns Run 2 configuration of HCAL noise filters to Run 1 mode

Martin-Grunewald committed:
Fix HLT Forward Port to 75X

davidlange6 committed:
Merge pull request #8822 from vkhristenko/CMSSW_7_5_X_HCALDQMuTCA

uTCA-related HCAL DQM Changes

davidlange6 committed:
Merge pull request #8867 from Martin-Grunewald/FixForwardPort

Fix HLT Forward Port to 75X

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

aledegano committed:
BigObjs apply conditions for slc7 too.
The special compiler flags are needed for slc7_amd64_gcc49* too.
The change is trivial and doesn't change anything for current production architecture.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

davidlange6 committed:
Merge pull request #8874 from degano/75X_bigobjs_slc7

BigObjs apply conditions for slc7 too.

davidlange6 committed:
Merge pull request #8859 from alja/roo6fw75

75 fireworks: port changes from 7_4_X, related to ROOT6 dictionaries

cmsbuild committed:
Merge pull request #8839 from dildick/GEM-static-analyzer-cleanup

Fix issues in GEM packages reported by static analyzer

venturia committed:
Merge branch 'calibtracker_sistripquality-75x' of github.com:venturia/cmssw into calibtracker_sistripquality-75x

Conflicts:
	CalibTracker/SiStripQuality/README.md

venturia commented:
I think I have to close this pull request because the changes it contains are already in 75x
